nodejs_bot_base/index.js

```diff
@@ -0,0 +1,10 @@
+const { deltachat, log } = require('deltachat-node-bot-base')
+
+// Start the deltachat core engine and handle incoming messages.
+deltachat.start((chat, message) => {
+  log("Got a message: ", message.getText())
+  if (deltachat.getChatContacts(chat.getid()).size === 1) {
```

Comment:
```suggestion
  if (deltachat.getChatContacts(chat.getid()).length === 1) {
```
[].size doesn't exist in javascript.